api/application/client_test.go

```diff
@@ -453,10 +456,84 @@ func (s *applicationSuite) TestDestroyConsumedApplicationsArity(c *gc.C) {
 	client := newClient(func(objType string, version int, id, request string, a, response interface{}) error {
 		return nil
 	})
-	_, err := client.DestroyConsumedApplication("foo")
+	destroyParams := application.DestroyConsumedApplicationParams{
+		[]string{"foo"}, false, nil,
+	}
+	_, err := client.DestroyConsumedApplication(destroyParams)
 	c.Assert(err, gc.ErrorMatches, `expected 1 result\(s\), got 0`)
 }
 
+func (s *applicationSuite) TestDestroyConsumedApplicationsForcev9(c *gc.C) {
+	var called bool
+	client := application.NewClient(basetesting.BestVersionCaller{
+		APICallerFunc: basetesting.APICallerFunc(
+			func(objType string, version int, id, request string, a, response interface{}) error {
+				called = true
+				return nil
```

Comment:
Yes, this is great and this is where we'd want to check that force and maxwait were not passed in.

Reply:
And to add the test that if the force and maxwait is passed in to an older version, we get the message that they are not supported.

Reply:
Sounds good, thanks for taking another look :)